lib/tasks/cleanup.rake

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+namespace :dpul do
+  namespace :cleanup do
+    desc "delete duplicate sidecars"
+    task sidecars: :environment do
```

Comment:
Did you decide on just a task vs having an object you can test, or is this a spike?

Reply:
I wouldn't say we decided but we sort of leaned toward just a task since we don't really know how we got into this mess to begin with. We thought we could test it first with the print statements.